.github/workflows/continuous-integration-unit-php.yml

```diff
@@ -40,7 +40,6 @@ jobs:
 
           - php: '7.4'
             wp: 'trunk'
-            experimental: true
```

Comment:
Don't we want to keep this?

You never know how trunk changes.

Reply:
Added it back.